decidim-core/spec/content_renderers/decidim/hashtag_renderer_spec.rb

```diff
@@ -94,5 +94,14 @@ module Decidim
         it { is_expected.to eq([hashtag]) }
       end
     end
+
+    describe "when content contains urls" do
+      let(:content) { %(Content with <a href="http://urls.net" onmouseover="alert(document.cookies)">URLs</a> of anchor type and text urls like https://decidim.org.) }
```

Comment:
Could you please add a test with a malicious `href` to make sure we don't allow them?